res/css/views/rooms/_ReplyPreview.scss

```diff
@@ -15,6 +15,9 @@ limitations under the License.
 */
 
 .mx_ReplyPreview {
+    --ReplyPreview_padding-inline: $spacing-16;
+    --ReplyPreview_gap-inline: $spacing-8;
```

Comment:
Not quite sure this change is needed in the scope of this pull request, what does it bring?

Reply:
For `max-width: calc(100% - var(--ReplyPreview_padding-inline) - 16px - var(--ReplyPreview_gap-inline));`